models/dbfs/dbfs.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package dbfs
+
+import (
+	"context"
+	"os"
+
+	"code.gitea.io/gitea/models/db"
+)
+
+type dbfsMeta struct {
+	ID              int64  `xorm:"pk autoincr"`
+	FullPath        string `xorm:"VARCHAR(500) UNIQUE NOT NULL"`
+	BlockSize       int64  `xorm:"BIGINT NOT NULL"`
+	FileSize        int64  `xorm:"BIGINT NOT NULL"`
+	CreateTimestamp int64  `xorm:"BIGINT NOT NULL"`
+	ModifyTimestamp int64  `xorm:"BIGINT NOT NULL"`
```

Comment:
```suggestion
	CreatedUnix timeutil.TimeStamp `xorm:"created"`
	UpdatedUnix timeutil.TimeStamp `xorm:"updated"`
```
Or is there a reason to deviate from the standard present throughout the rest of the application?

Reply:
The `created` and `updated` in `xorm` are designed for recording the times of db raw. However, the `CreateTimestamp` and `ModifyTimestamp` are for the virtual file, they could be different from the times of db raw. Just like we can use `touch` command to change the modification or creation time of a file in disk.